### ISSUE TITLE
Symbol to trim thread can be digitized

### ISSUE THREAD

**Issue by bkmgit** [inkstitch/inkstitch#2050]
When using Inkscape 1.2.2 (b0a8486541, 2022-12-01) on Fedora Linux, with Ink/Stitch v2.1.2 the symbol to trim the thread can become a full part of the SVG file that is then digitized and would be embroidered. 

An example such file is attached:

| File with Error | Correct File |
|---                        |---                   |
| ![ErrorExample](https://user-images.githubusercontent.com/1679477/216755705-50d960ed-9fbd-45e7-92f3-540019023fca.png) | ![CorrectExample](https://user-images.githubusercontent.com/1679477/216756254-95b0b847-3752-43bc-9770-975d1d7e75ab.png) |
| ![ErrorExample](https://user-images.githubusercontent.com/1679477/216755795-3ef4dc5c-7124-42d7-b40d-bff692eda013.svg) | ![CorrectExample](https://user-images.githubusercontent.com/1679477/216756289-64d705e3-6533-4839-86de-1ac74d695c66.svg)|

**Comment by lexelby** [comment]
Are you saying that Ink/Stitch produced the file on the left?  That looks like maybe "Object to Path" was applied, maybe?

**Comment by tyrosinase** [comment]
> Are you saying that Ink/Stitch produced the file on the left? That looks like maybe "Object to Path" was applied, maybe?

This happens for me... well, not quite *frequently*, but often enough when trying to copy/import/etc. objects with commands attached. I am pretty sure I nailed down the conditions that time I tried all those combos, but I don't know where that thread is offhand.

I don't think I looked closely at the object afterward though. I sort of assumed it was still a Circle.
